sdk/python/jobs/automl-standalone-jobs/automl-forecasting-demand-forecasting-many-models/auto-ml-forecasting-demand-forecasting-many-models.ipynb

```diff
@@ -0,0 +1,1429 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "endpoint = BatchEndpoint(\n",
+    "    name=endpoint_name,\n",
+    "    description=\"A many models endpoint for component deployments\",\n",
+    "    properties={\"ComponentDeployment.Enabled\": True},\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following command creates the Endpoint in the workspace usign the MLClient created earlier. This command will start the endpoint creation and return a confirmation response while the endpoint creation continues."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.batch_endpoints.begin_create_or_update(endpoint).result()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 10.2. Create the Deployment\n",
+    "\n",
+    "A deployment is a set of resources required for hosting the model that does the actual inferencing."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "deployment = BatchDeployment(\n",
+    "    name=\"sdk-many-models-deployment\",\n",
+    "    description=\"A many models deployment.\",\n",
+    "    endpoint_name=endpoint_name,\n",
+    "    compute=amlcompute_cluster_name,\n",
+    "    job_definition=JobDefinition(\n",
+    "        type=\"pipeline\",\n",
+    "        component=inference_component_from_registry.id,\n",
+    "        settings={\"compute\": amlcompute_cluster_name},\n",
+    "    ),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following command creates the deployment in the workspace usign the MLClient created earlier. This command will start the deployment creation and return a confirmation response while the deployment creation continues."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.batch_deployments.begin_create_or_update(deployment).result()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 10.3. Invoke the Endpoint\n",
+    "\n",
+    "The next cell contians the command that invokes the endpoint for batch inference job. The `invoke` method contains the `inputs` parameter. This parameter contains the inputs necessary to execute the inference component on the endpoint. To convince yourself this is the case, compare the input parameters for the `inference_component_from_registry` in section 6.3 with the `inputs` we are proving in the next cell. They are identical.\n",
+    "\n",
+    "Notice, the the `forecast_mode` is set to `\"recursive\"`. In the evaluation pipeline this component was used to generate rolling forecast to evalaute model performance on the test set. Here, we are using it to generate a forecast."
```

Comment:
Shall we include a link to the documentation which explains rolling/recursive forecast?

Reply:
Done. Included a placeholder link to Eric's documentation. Will update once it is published.